libs2ecore/src/S2EExecutor.cpp

```diff
@@ -1370,14 +1370,14 @@ Executor::StatePair S2EExecutor::forkAndConcretize(S2EExecutionState *state, kle
 
 S2EExecutor::StatePair S2EExecutor::fork(ExecutionState &current, const klee::ref<Expr> &condition,
                                          bool keepConditionTrueInCurrentState) {
-    return doFork(current, &condition, keepConditionTrueInCurrentState);
+    return doFork(current, condition, keepConditionTrueInCurrentState);
 }
 
 S2EExecutor::StatePair S2EExecutor::fork(ExecutionState &current) {
     return doFork(current, nullptr, false);
```

Comment:
In PR #34, this new version of fork() was added, allowing the user to fork without providing a condition. After careful consideration, the second argument `nullptr` here results in a construction of ref<Expr> with a nullptr, and this ref<Expr> will be passed by const reference to doFork().